cmd/root.go

```diff
@@ -34,19 +37,23 @@ func Execute() {
 func init() {
 	cobra.OnInitialize(initConfig)
 
-	RootCmd.PersistentFlags().StringVar(&sfdcUsername, "username", "", "Salesforce Username")
-	RootCmd.PersistentFlags().StringVar(&sfdcPassword, "password", "", "Salesforce Password")
-	RootCmd.PersistentFlags().StringVar(&appClientID, "client-id", "", "Connected App Client ID")
-	RootCmd.PersistentFlags().StringVar(&appClientSecret, "client-secret", "", "Connected App Client Secret")
-	RootCmd.PersistentFlags().StringVar(&apiVersion, "api-version", "36.0", "Salesforce API Version")
-	RootCmd.PersistentFlags().StringVar(&sfdcHost, "host", "https://login.salesforce.com", "Salesforce Login Url")
-
-	if sfdcUsername == "" {
-		sfdcUsername = os.Getenv("SKUID_UN")
+	RootCmd.PersistentFlags().StringVar(&appClientID, "client-id", "", "OAuth Client ID")
+	RootCmd.PersistentFlags().StringVar(&appClientSecret, "client-secret", "", "OAuth Client Secret")
+	RootCmd.PersistentFlags().StringVar(&apiVersion, "api-version", "", "API Version")
```

Comment:
i don't think this is necessary - you're already using environment variables to control this, no need for multiple flags in my opinion